com.github.finefindus.eyedropper.json

```diff
@@ -0,0 +1,32 @@
+{
+    "id": "com.github.finefindus.eyedropper",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "eyedropper",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=RUST_LOG=eyedropper=debug",
```

Comment:
debug logging?

Reply:
oops, I forgot to remove it. It's removed now.